eclipse-jdt/eclipse.jdt.ui
# dogfooding - if else to switch cleanup

## What it does
Applying the if/else to switch cleanup to the codebase because two additional cases are supported now.
Not in all cases the code is much clearer after applying the cleanup. However in these cases the code is definitely not very clear before as well. So in real world usages the feature to switch off the cleanup on specific code locations comes in handy.

## How to test
It should not change the way the code is processing. Might be in some corner cases switch is faster than if else chains.

Note:
There was a bug when processing "JUnitClasspathFixProcessor.java":
The comments for the strings have been removed and not reapplied. Not sure if that is a big deal as it can be applied automatically in a second step.

## Author checklist
